File: v2/jaxrs/src/main/java/io/swagger/model/Memberresponse.java
package io.swagger.model;

import io.swagger.model.Member;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Memberresponse  {
  
  private Member post = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("post")
  public Member getPost() {
    return post;
  }
  public void setPost(Member post) {
    this.post = post;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Memberresponse {\n");
    
    sb.append("  post: ").append(post).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/io/swagger/Model/Memberresponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Memberresponse {
    

    
    public Member Member { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Memberresponse {\n");
      
      sb.Append("  Member: ").Append(Member).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}